plugins/rapid7_vulndb/plugin.spec.yaml

```diff
@@ -19,7 +19,8 @@ key_features:
   - Search Database for vulnerabilities
   - Retrieve vulnerability or Metasploit module details based on a CVE or module identifier
 requirements: "_This plugin does not contain any requirements._"
-troubleshooting: "_This plugin does not contain any troubleshooting information._"
+troubleshooting:
+  - "This plugin does not contain any troubleshooting information."
```

Comment:
Does the troubleshooting as a list of elements instead of string work with `insight-plugin` refresh? :DD